internal/adapters/operation/operation_storage_redis.go

```diff
@@ -365,3 +365,28 @@ func buildOperationFromMap(opMap map[string]string) (*operation.Operation, error
 		ExecutionHistory: executionHistory,
 	}, nil
 }
+
+func (r *redisOperationStorage) CleanExpiredOperations(ctx context.Context, schedulerName string) error {
+	operationsIDs, err := r.client.ZRangeByScore(ctx, r.buildSchedulerHistoryOperationsKey(schedulerName), &redis.ZRangeBy{
+		Min: "-inf",
+		Max: "+inf",
+	}).Result()
+
+	pipe := r.client.Pipeline()
+
+	if len(operationsIDs) == 0 {
+		for _, operationID := range operationsIDs {
```

Comment:
@mflilian Before removing the operation from sorted set, you must verify if this key exists first

Reply:
just did it sir, thanks